src/Kunstmaan/NodeBundle/Controller/NodeAdminController.php

```diff
@@ -126,26 +126,7 @@ public function indexAction(Request $request): Response
     {
         $this->init($request);
 
-        $nodeAdminListConfigurator = new NodeAdminListConfigurator(
-            $this->em,
-            $this->aclHelper,
-            $this->locale,
-            PermissionMap::PERMISSION_VIEW,
-            $this->authorizationChecker
-        );
-
-        $locale = $this->locale;
-        $acl = $this->authorizationChecker;
-        $itemRoute = function (EntityInterface $item) use ($locale, $acl) {
-            if ($acl->isGranted(PermissionMap::PERMISSION_VIEW, $item->getNode())) {
-                return [
-                    'path' => '_slug_preview',
-                    'params' => ['_locale' => $locale, 'url' => $item->getUrl()],
-                ];
-            }
-        };
-        $nodeAdminListConfigurator->addSimpleItemAction('action.preview', $itemRoute, 'eye');
```

Comment:
This code is not added in `getAdminListConfigurator` but is still needed I think?

Reply:
See https://github.com/Kunstmaan/KunstmaanBundlesCMS/pull/3334/files#diff-ec9c9d7d2b1ae0e92e0c5bee37ebb65e41a8abcd9a95c9eae80665cf998c30f7R132 or do you mean something else?

Reply:
Oh I have missed it somehow, all good!